specifications/api/swagger.yaml

```diff
@@ -35,12 +35,27 @@ numberOfSlots: &numberOfSlots
   properties:
     quantity:
       type: integer
-      minimum: 0
+      minimum: 1337
```

Comment:
a bit more "arbitrary" number

Reply:
Hmm... But this is wrong here, it states that the minimum should be 1337 :no_mouth: 
Did you mean to add it as `example` ?